west.yml

```diff
@@ -241,15 +241,20 @@ manifest:
       groups:
         - tee
     - name: tf-m-tests
-      revision: bcb53bccccdc05c713aade707e7a8ddad35c210f
+      revision: pull/5/head
       path: modules/tee/tf-m/tf-m-tests
       groups:
         - tee
     - name: psa-arch-tests
-      revision: f4fc2442b8e29e2a03d9899e46e5a3ea3df8c2c9
+      revision: pull/7/head
       path: modules/tee/tf-m/psa-arch-tests
       groups:
         - tee
+    - name: qcbor
+      revision: b0e7033268e88c9f27146fa9a1415ef4c19ebaff
+      path: modules/tee/tf-m/qcbor
```

Comment:
Looks good. Just remove this as discussed in TSC meeting today.

Reply:
Removed commit. Thank you.